src/v/coproc/router.cc

```diff
@@ -36,7 +40,7 @@ router::extract_offset(model::record_batch_reader reader) {
           if (data.empty()) {
               return std::optional<offset_rbr_pair>(std::nullopt);
           }
-          const auto last_offset = data.back().last_offset();
+          const model::offset last_offset = data.back().last_offset();
```

Comment:
is this necessary?

Reply:
it isn't necessary but i thought adding the type for self-documentation purposes was a good idea

Reply:
sounds good. it's not a big deal, it just looked random.